lib/which_formula.rb

```diff
@@ -26,7 +26,7 @@ def download_and_cache_executables_file!(skip_update: false)
         retries = Homebrew::EnvConfig.curl_retries.to_i
       else
         max_time = 10 # seconds
-        retries = 1 # do not retry by default
+        retries = 0 # do not retry by default
```

Comment:
```suggestion
        retries = nil # do not retry by default
```

Reply:
Setting to `0` is equivalent to `nil` according to the current state of the `brew` upstream. And `retries = 0` (no retry) takes even longer than `retries = 1` (retry once) as I mentioned in https://github.com/Homebrew/brew/pull/20722#issue-3429098040.